python/taichi/lang/__init__.py

```diff
@@ -121,19 +121,29 @@ def init(arch=None,
     for k, v in kwargs.items():
         setattr(ti.cfg, k, v)
 
-    def boolean_config(key, name=None):
-        if name is None:
-            name = 'TI_' + key.upper()
+    def boolean_config(key):
```

Comment:
It seems to me that this function is never used. Please remove. Again, please proof-read your code 3-5 times before requesting a review to avoid abuse of the community resources.